### PR TITLE
Project README update

### PR DESCRIPTION
**Pull request by Hamled**
This project README currently includes:

- Brief project overview
- Learning goals
- Basic setup and REPL instructions
- Complete requirements for wave one
- Brief explanation of waves two and three

The project README still needs to have:
- Detailed requirements for waves two and three

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,89 @@
...
+
+This might seem like an unwarranted distraction from diving right into the code when you're under such time constraints. But a great deal of time can be saved by avoiding costly refactors if it turns out that your initial approach won't allow you to achieve certain requirements. The more complex the application's logic is, the more difficult and time-consuming it will be to rework that logic after it has been written.
+
+#### User Stories
+- As a user of the Tic-Tac-Toe game I CAN:
+  - Create a new game:
+    - It will have an empty board of nine cells
+    - It will start with the X player having the first move
+  - Specify up to two player names
+  - Get the current outcome of the game (X win, O win, tie, or undecided)
```
Comment (kariabancroft) [inline README.md:50]: What would be the difference between tie and undecided?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,89 @@
...
+  - (OPTIONAL) View a printed version of the current game board for debugging purposes
+- As the 'active' player in a game I CAN:
```
Comment (kariabancroft) [inline README.md:52]: This is a little bit confusing about having a separate "player" object, or just having this logic in a part of the game. Is there a way we can help clarify this part?

Comment (kariabancroft) [reply README.md:52]: Nevermind, I see your clarifying points later on which seem to help

**Comment by PilgrimMemoirs** [comment]
This looks good (thanks Charles!), However i do think it 'gives away' a lot of that design. I think I'm going to do the problem solving portion first, where they have to think about these things on their own, then merge this for them to use as they do build.

I think they're all important things they need to be aware of, but they need some time to think of what those user stories and that specific functionality before handing it to them.

File: README.md (hunk 1 above)
Comment (Hamled) [reply README.md:50]: Undecided means the game is not complete. I will change the wording to say incomplete.

**Pull request merged**